doc/specs/index.md

```diff
@@ -14,6 +14,8 @@ This is and index/directory of the specifications (specs) for each new module/fe
  - [ascii](./stdlib_ascii.html) - Procedures for handling ASCII characters
  - [bitsets](./stdlib_bitsets.html) - Bitset data types and procedures
  - [error](./stdlib_error.html) - Catching and handling errors
+ - [hash\_functions](./stdlib_has_functions.html) - Hashing integer
```

Comment:
Possible typo? ( `_has_functions` instead of `_hash_functions` ?).

Reply:
Yes it is a typo.